draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+
+
+
+
+TEAS Working Group                                                X. Liu
+Internet-Draft                                           IBM Corporation
+Intended status: Standards Track                             J. Tantsura
+Expires: 13 August 2023                                        Microsoft
+                                                              I. Bryskin
+                                                              Individual
+                                                          L.M. Contreras
+                                                              Telefonica
+                                                                   Q. Wu
+                                                                  Huawei
+                                                              S. Belotti
+                                                                   Nokia
+                                                                R. Rokui
+                                                                   Ciena
+                                                                  A. Guo
+                                                               Futurewei
+                                                                 I. Busi
+                                                                  Huawei
+                                                         9 February 2023
+
+
+              IETF Network Slice Topology YANG Data Model
+             draft-liu-teas-transport-network-slice-yang-06
+
+Abstract
+
+   An IETF network slice may use an abstract topology to describe
+   intended underlay for connectivities between slice endpoints.
+   Abstract topologies help the customer to configure network slices
+   with shared resources amongst connections, and connections can be
```

Comment:
s/ customer to configure network slices / customer to request network slices

Reply:
ok to change it to "request"